ASPNetCoreMastersTodoList.Api/Controllers/ItemsController.cs

```diff
@@ -0,0 +1,25 @@
+﻿using ASPNetCoreMastersTodoList.Api.BindingModels;
+using Microsoft.AspNetCore.Mvc;
+using Services.DTO;
+using Services.Interfaces;
+
+namespace ASPNetCoreMastersTodoList.Controllers
+{
+    public class ItemsController : ControllerBase
+    {
+        private readonly IItemService _itemService;
+        public ItemsController(IItemService itemService)
+        {
+            _itemService = itemService;
+        }
+        public int Get(int id)
+        {
+            return id;
+        }
+
+        public void Post(ItemCreateApiModel model)
+        {
+            _itemService.Save(new ItemDto { Item = model.Item });
```

Comment:
We need to check for model validation result using ModelState.IsValid to react to validation errors

Reply:
updated